README.md

```diff
@@ -156,11 +156,31 @@ Once you've done that, upload the new database using these commands:
 
 `psql -f databasename /travis/test.db`
 
-There shouldn't be any errors. If you log into the database using 
+There shouldn't be any errors. If you log into the database using
 `psql databasename`
 you should be able to run SQL queries to look at all of the information on the
 database. For example, `\dt` should list the available tables.
 
+## Install Sass and Compile into CSS
+
+NOTE: You will only need to install Sass if you're going to make front-end style changes.
+(Without Sass, you can still use Bootstrap classes and our classes in HTML, but you won't be able to manipulate them or make your own.
+This also means you SHOULD NOT EDIT `styles.css`; it is a generated file, and your changes will be overwritten by Sass users.)
+
+Go to [http://sass-lang.com/install](http://sass-lang.com/install) and install Sass for your system. Installing for the command line is recommended.
+
+When you have Sass installed, you can run it to compile a CSS file.
+The output should ALWAYS be to `server/static/styles.css` until further notice.
+The file `server/assets/scss/index.scss` should always be the input file and the only input file, as it imports the rest of the .scss files.
+
+As an example, here is the command for Linux:
+
+`sass --watch server/assets/scss/index.scss server/static/styles.css`
+
+Using a watch command is recommended so that changes to .scss files are automatically recompiled.
+
+Even with the watch command, Flask won't update `styles.css` on the page, so you'll have to refresh and probably clear the cache too (CTRL + F5).
+
```

Comment:
Thank you for including documentation on new technology without having to be asked!!! You're the best!